_includes/configuration/image-file-types.md

```diff
@@ -0,0 +1,20 @@
+### `image_file_types`
+
+This option configures which image file formats will be recognised and modified by the [smart_paste](#smart_paste) functionality when content is pasted into the editor.
```

Comment:
```suggestion
This option configures which image file formats will be recognized and modified by the [`smart_paste`](#smart_paste) functionality when content is pasted into the editor.
```
US spelling. 
Also, please mention what it's doing with these file formats?

Reply:
The `smart_paste` docs explain what happens, should I reiterate it here?

Reply:
a little. It's not clear that this option determines which converts urls to images... all content needs to be able to stand on it's own to some extent, as we can't control how people will enter the docs, read the docs, or how much they will read.